File: hosting/files/Quality_of_Life_Game_20210301.cs
/* 
*  DB Script Tool
*  C# Entity Framework (C#) - 2021-03-01 14:48:58
*  
*  MODEL CLASSES FOR Quality_of_Life_Game DATABASE
*/



/* Player.cs -------------------------------------*/
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Data.Entity;

namespace Quality_of_Life_Game.Models 
{

    /// <summary>
    ///  C# - Model Class - Quality_of_Life_Game.Player
    ///  2021-03-01 13:42:44
    /// </summary>
    //[Table("Player")]
    public class Player {

       /// <summary>
       ///  Constructor
       /// </summary>
       /// <example>
       ///   Example
       ///   <code>
       ///   Player myPlayer = new Player();
       ///   </code>
       /// </example>
        public Player() {
            //--
            this.PlayerList = new HashSet<Player>();
        }

       /// <summary>
       ///  Constructor
       /// </summary>
       /// <example>
       ///   Example
       ///   <code>
       ///   Player myPlayer = new Player( val1, val2,.. );
       ///   </code>
       /// </example>
        public Player(int id, long redrace, long yellowrace, long purplerace, long greenrace, long orangerace, long bluerace) {
            this.Id = id;
            this.Redrace = redrace;
            this.Yellowrace = yellowrace;
            this.Purplerace = purplerace;
            this.Greenrace = greenrace;
            this.Orangerace = orangerace;
            this.Bluerace = bluerace;
            this.PlayerList = new HashSet<Player>();
        }


        /**
        * Properties
        */

        [Key]
        [DatabaseGeneratedAttribute(DatabaseGeneratedOption.Identity)]
        public int Id { get; set; }

        public long Redrace { get; set; }

        public long Yellowrace { get; set; }

        public long Purplerace { get; set; }

        public long Greenrace { get; set; }

        public long Orangerace { get; set; }

        public long Bluerace { get; set; }



        public virtual ICollection<Player> PlayerList { get; set; }



        /**
        * Methods
        */

        public override string ToString() {
            return "";
        }

    }

}



/* Nonplayer.cs -------------------------------------*/
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Data.Entity;

namespace Quality_of_Life_Game.Models 
{

    /// <summary>
    ///  C# - Model Class - Quality_of_Life_Game.Nonplayer
    ///  2021-03-01 13:56:27
    /// </summary>
    //[Table("Nonplayer")]
    public class Nonplayer {

       /// <summary>
       ///  Constructor
       /// </summary>
       /// <example>
       ///   Example
       ///   <code>
       ///   Nonplayer myNonplayer = new Nonplayer();
       ///   </code>
       /// </example>
        public Nonplayer() {
            //--
        }

       /// <summary>
       ///  Constructor
       /// </summary>
       /// <example>
       ///   Example
       ///   <code>
       ///   Nonplayer myNonplayer = new Nonplayer( val1, val2,.. );
       ///   </code>
       /// </example>
        public Nonplayer(int _id, int rexoedipus, int lightbluerace, int pinkrace, int brownrace, int fred, int edtdoide, int genecrispy, int hysteriaptrance, int officermodusponens, int officermodustollens, int sifidious, int imhotepbizarros, int barryukolmar) {
            this._id = _id;
            this.Rexoedipus = rexoedipus;
            this.Lightbluerace = lightbluerace;
            this.Pinkrace = pinkrace;
            this.Brownrace = brownrace;
            this.Fred = fred;
            this.Edtdoide = edtdoide;
            this.Genecrispy = genecrispy;
            this.Hysteriaptrance = hysteriaptrance;
            this.Officermodusponens = officermodusponens;
            this.Officermodustollens = officermodustollens;
            this.Sifidious = sifidious;
            this.Imhotepbizarros = imhotepbizarros;
            this.Barryukolmar = barryukolmar;
        }


        /**
        * Properties
        */

        [Key]
        [DatabaseGeneratedAttribute(DatabaseGeneratedOption.Identity)]
        public int _id { get; set; }

        public int Rexoedipus { get; set; }

        public int Lightbluerace { get; set; }

        public int Pinkrace { get; set; }

        public int Brownrace { get; set; }

        public int Fred { get; set; }

        public int Edtdoide { get; set; }

        public int Genecrispy { get; set; }

        public int Hysteriaptrance { get; set; }

        public int Officermodusponens { get; set; }

        public int Officermodustollens { get; set; }

        public int Sifidious { get; set; }

        public int Imhotepbizarros { get; set; }

        public int Barryukolmar { get; set; }





        /**
        * Methods
        */

        public override string ToString() {
            return "";
        }

    }

}



/* Neighborhoods.cs -------------------------------------*/
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Data.Entity;

namespace Quality_of_Life_Game.Models 
{

    /// <summary>
    ///  C# - Model Class - Quality_of_Life_Game.Neighborhoods
    ///  2021-03-01 14:19:40
    /// </summary>
    //[Table("Neighborhoods")]
    public class Neighborhoods {

       /// <summary>
       ///  Constructor
       /// </summary>
       /// <example>
       ///   Example
       ///   <code>
       ///   Neighborhoods myNeighborhoods = new Neighborhoods();
       ///   </code>
       /// </example>
        public Neighborhoods() {
            //--
        }

       /// <summary>
       ///  Constructor
       /// </summary>
       /// <example>
       ///   Example
       ///   <code>
       ///   Neighborhoods myNeighborhoods = new Neighborhoods( val1, val2,.. );
       ///   </code>
       /// </example>
        public Neighborhoods(int _id, int redneighborhood, int yellowneighborhood, int purpleneigborhood, int greenneighborhood, int orangeneighborhood, int blueneighborhood) {
            this._id = _id;
            this.Redneighborhood = redneighborhood;
            this.Yellowneighborhood = yellowneighborhood;
            this.Purpleneigborhood = purpleneigborhood;
            this.Greenneighborhood = greenneighborhood;
            this.Orangeneighborhood = orangeneighborhood;
            this.Blueneighborhood = blueneighborhood;
        }


        /**
        * Properties
        */

        [Key]
        [DatabaseGeneratedAttribute(DatabaseGeneratedOption.Identity)]
        public int _id { get; set; }

        //[ForeignKey("Player")]
        public int Redneighborhood { get; set; }
        public virtual Player Player { get; set; }

        //[ForeignKey("Player")]
        public int Yellowneighborhood { get; set; }
        public virtual Player Player { get; set; }

        public int Purpleneigborhood { get; set; }

        public int Greenneighborhood { get; set; }

        public int Orangeneighborhood { get; set; }

        public int Blueneighborhood { get; set; }





        /**
        * Methods
        */

        public override string ToString() {
            return "";
        }

    }

}



/* Social_Bubbles.cs -------------------------------------*/
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Data.Entity;

namespace Quality_of_Life_Game.Models 
{

    /// <summary>
    ///  C# - Model Class - Quality_of_Life_Game.Social_Bubbles
    ///  
    /// </summary>
    //[Table("Social_Bubbles")]
    public class Social_Bubbles {

       /// <summary>
       ///  Constructor
       /// </summary>
       /// <example>
       ///   Example
       ///   <code>
       ///   Social_Bubbles mySocial_Bubbles = new Social_Bubbles();
       ///   </code>
       /// </example>
        public Social_Bubbles() {
            //--
        }

       /// <summary>
       ///  Constructor
       /// </summary>
       /// <example>
       ///   Example
       ///   <code>
       ///   Social_Bubbles mySocial_Bubbles = new Social_Bubbles( val1, val2,.. );
       ///   </code>
       /// </example>
        public Social_Bubbles(int _id, int socioeconomicstatus, int household, int race, int transportation, int epidemiologicalfactors, int healthfactors) {
            this._id = _id;
            this.Socioeconomicstatus = socioeconomicstatus;
            this.Household = household;
            this.Race = race;
            this.Transportation = transportation;
            this.Epidemiologicalfactors = epidemiologicalfactors;
            this.Healthfactors = healthfactors;
        }


        /**
        * Properties
        */

        [Key]
        [DatabaseGeneratedAttribute(DatabaseGeneratedOption.Identity)]
        public int _id { get; set; }

        public int Socioeconomicstatus { get; set; }

        public int Household { get; set; }

        public int Race { get; set; }

        public int Transportation { get; set; }

        public int Epidemiologicalfactors { get; set; }

        public int Healthfactors { get; set; }





        /**
        * Methods
        */

        public override string ToString() {
            return "";
        }

    }

}



/* Health_Factors.cs -------------------------------------*/
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Data.Entity;

namespace Quality_of_Life_Game.Models 
{

    /// <summary>
    ///  C# - Model Class - Quality_of_Life_Game.Health_Factors
    ///  
    /// </summary>
    //[Table("Health_Factors")]
    public class Health_Factors {

       /// <summary>
       ///  Constructor
       /// </summary>
       /// <example>
       ///   Example
       ///   <code>
       ///   Health_Factors myHealth_Factors = new Health_Factors();
       ///   </code>
       /// </example>
        public Health_Factors() {
            //--
        }

       /// <summary>
       ///  Constructor
       /// </summary>
       /// <example>
       ///   Example
       ///   <code>
       ///   Health_Factors myHealth_Factors = new Health_Factors( val1, val2,.. );
       ///   </code>
       /// </example>
        public Health_Factors(int _id, int unexpectedpregnancy, int measles, int mumps, int rubella, int pneumonia, int diabetes) {
            this._id = _id;
            this.Unexpectedpregnancy = unexpectedpregnancy;
            this.Measles = measles;
            this.Mumps = mumps;
            this.Rubella = rubella;
            this.Pneumonia = pneumonia;
            this.Diabetes = diabetes;
        }


        /**
        * Properties
        */

        [Key]
        [DatabaseGeneratedAttribute(DatabaseGeneratedOption.Identity)]
        public int _id { get; set; }

        public int Unexpectedpregnancy { get; set; }

        public int Measles { get; set; }

        public int Mumps { get; set; }

        public int Rubella { get; set; }

        public int Pneumonia { get; set; }

        public int Diabetes { get; set; }





        /**
        * Methods
        */

        public override string ToString() {
            return "";
        }

    }

}



/* Socioeconomic_Status.cs -------------------------------------*/
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Data.Entity;

namespace Quality_of_Life_Game.Models 
{

    /// <summary>
    ///  C# - Model Class - Quality_of_Life_Game.Socioeconomic_Status
    ///  2021-03-01 14:36:28
    /// </summary>
    //[Table("Socioeconomic_Status")]
    public class Socioeconomic_Status {

       /// <summary>
       ///  Constructor
       /// </summary>
       /// <example>
       ///   Example
       ///   <code>
       ///   Socioeconomic_Status mySocioeconomic_Status = new Socioeconomic_Status();
       ///   </code>
       /// </example>
        public Socioeconomic_Status() {
            //--
        }

       /// <summary>
       ///  Constructor
       /// </summary>
       /// <example>
       ///   Example
       ///   <code>
       ///   Socioeconomic_Status mySocioeconomic_Status = new Socioeconomic_Status( val1, val2,.. );
       ///   </code>
       /// </example>
        public Socioeconomic_Status(int _id, int bankruptcy, int unemployment, int homeinvasion, int incomedrop, int incomeloss, int eviction) {
            this._id = _id;
            this.Bankruptcy = bankruptcy;
            this.Unemployment = unemployment;
            this.Homeinvasion = homeinvasion;
            this.Incomedrop = incomedrop;
            this.Incomeloss = incomeloss;
            this.Eviction = eviction;
        }


        /**
        * Properties
        */

        [Key]
        [DatabaseGeneratedAttribute(DatabaseGeneratedOption.Identity)]
        public int _id { get; set; }

        public int Bankruptcy { get; set; }

        public int Unemployment { get; set; }

        public int Homeinvasion { get; set; }

        public int Incomedrop { get; set; }

        public int Incomeloss { get; set; }

        public int Eviction { get; set; }





        /**
        * Methods
        */

        public override string ToString() {
            return "";
        }

    }

}

File: hosting/files/Quality_of_Life_Game_20210301.py
#  
#  DB Script Tool
#  Python - 2021-03-01 14:48:38
#  
#  MODEL CLASSES FOR Quality_of_Life_Game DATABASE
#



# Player.py -------------------------------------
from datetime import datetime, date

# 
# Python - Model Class - Quality_of_Life_Game.Player
# 2021-03-01 13:42:44
#
class Player(Object):

    #
    # Constructor
    #
    # Example: 
    # myPlayer = Player( val1, val2,.. )
    #
    def __init__(self, id = None, redrace = None, yellowrace = None, purplerace = None, greenrace = None, orangerace = None, bluerace = None):
        self.__id = id
        self.__redrace = redrace
        self.__yellowrace = yellowrace
        self.__purplerace = purplerace
        self.__greenrace = greenrace
        self.__orangerace = orangerace
        self.__bluerace = bluerace


    #
    # Properties
    #

    @property
    def id(self):
        return self.__id

    @id.setter
    def id(self, id):
        self.__id = id

    @property
    def redrace(self):
        return self.__redrace

    @redrace.setter
    def redrace(self, redrace):
        self.__redrace = redrace

    @property
    def yellowrace(self):
        return self.__yellowrace

    @yellowrace.setter
    def yellowrace(self, yellowrace):
        self.__yellowrace = yellowrace

    @property
    def purplerace(self):
        return self.__purplerace

    @purplerace.setter
    def purplerace(self, purplerace):
        self.__purplerace = purplerace

    @property
    def greenrace(self):
        return self.__greenrace

    @greenrace.setter
    def greenrace(self, greenrace):
        self.__greenrace = greenrace

    @property
    def orangerace(self):
        return self.__orangerace

    @orangerace.setter
    def orangerace(self, orangerace):
        self.__orangerace = orangerace

    @property
    def bluerace(self):
        return self.__bluerace

    @bluerace.setter
    def bluerace(self, bluerace):
        self.__bluerace = bluerace



    #
    # Methods
    #

    def __str__ (self):
        return ""



# Nonplayer.py -------------------------------------
from datetime import datetime, date

# 
# Python - Model Class - Quality_of_Life_Game.Nonplayer
# 2021-03-01 13:56:27
#
class Nonplayer(Object):

    #
    # Constructor
    #
    # Example: 
    # myNonplayer = Nonplayer( val1, val2,.. )
    #
    def __init__(self, _id = None, rexoedipus = None, lightbluerace = None, pinkrace = None, brownrace = None, fred = None, edtdoide = None, genecrispy = None, hysteriaptrance = None, officermodusponens = None, officermodustollens = None, sifidious = None, imhotepbizarros = None, barryukolmar = None):
        self.___id = _id
        self.__rexoedipus = rexoedipus
        self.__lightbluerace = lightbluerace
        self.__pinkrace = pinkrace
        self.__brownrace = brownrace
        self.__fred = fred
        self.__edtdoide = edtdoide
        self.__genecrispy = genecrispy
        self.__hysteriaptrance = hysteriaptrance
        self.__officermodusponens = officermodusponens
        self.__officermodustollens = officermodustollens
        self.__sifidious = sifidious
        self.__imhotepbizarros = imhotepbizarros
        self.__barryukolmar = barryukolmar


    #
    # Properties
    #

    @property
    def _id(self):
        return self.___id

    @_id.setter
    def _id(self, _id):
        self.___id = _id

    @property
    def rexoedipus(self):
        return self.__rexoedipus

    @rexoedipus.setter
    def rexoedipus(self, rexoedipus):
        self.__rexoedipus = rexoedipus

    @property
    def lightbluerace(self):
        return self.__lightbluerace

    @lightbluerace.setter
    def lightbluerace(self, lightbluerace):
        self.__lightbluerace = lightbluerace

    @property
    def pinkrace(self):
        return self.__pinkrace

    @pinkrace.setter
    def pinkrace(self, pinkrace):
        self.__pinkrace = pinkrace

    @property
    def brownrace(self):
        return self.__brownrace

    @brownrace.setter
    def brownrace(self, brownrace):
        self.__brownrace = brownrace

    @property
    def fred(self):
        return self.__fred

    @fred.setter
    def fred(self, fred):
        self.__fred = fred

    @property
    def edtdoide(self):
        return self.__edtdoide

    @edtdoide.setter
    def edtdoide(self, edtdoide):
        self.__edtdoide = edtdoide

    @property
    def genecrispy(self):
        return self.__genecrispy

    @genecrispy.setter
    def genecrispy(self, genecrispy):
        self.__genecrispy = genecrispy

    @property
    def hysteriaptrance(self):
        return self.__hysteriaptrance

    @hysteriaptrance.setter
    def hysteriaptrance(self, hysteriaptrance):
        self.__hysteriaptrance = hysteriaptrance

    @property
    def officermodusponens(self):
        return self.__officermodusponens

    @officermodusponens.setter
    def officermodusponens(self, officermodusponens):
        self.__officermodusponens = officermodusponens

    @property
    def officermodustollens(self):
        return self.__officermodustollens

    @officermodustollens.setter
    def officermodustollens(self, officermodustollens):
        self.__officermodustollens = officermodustollens

    @property
    def sifidious(self):
        return self.__sifidious

    @sifidious.setter
    def sifidious(self, sifidious):
        self.__sifidious = sifidious

    @property
    def imhotepbizarros(self):
        return self.__imhotepbizarros

    @imhotepbizarros.setter
    def imhotepbizarros(self, imhotepbizarros):
        self.__imhotepbizarros = imhotepbizarros

    @property
    def barryukolmar(self):
        return self.__barryukolmar

    @barryukolmar.setter
    def barryukolmar(self, barryukolmar):
        self.__barryukolmar = barryukolmar



    #
    # Methods
    #

    def __str__ (self):
        return ""



# Neighborhoods.py -------------------------------------
from datetime import datetime, date

# 
# Python - Model Class - Quality_of_Life_Game.Neighborhoods
# 2021-03-01 14:19:40
#
class Neighborhoods(Object):

    #
    # Constructor
    #
    # Example: 
    # myNeighborhoods = Neighborhoods( val1, val2,.. )
    #
    def __init__(self, _id = None, redneighborhood = None, yellowneighborhood = None, purpleneigborhood = None, greenneighborhood = None, orangeneighborhood = None, blueneighborhood = None):
        self.___id = _id
        self.__redneighborhood = redneighborhood
        self.__yellowneighborhood = yellowneighborhood
        self.__purpleneigborhood = purpleneigborhood
        self.__greenneighborhood = greenneighborhood
        self.__orangeneighborhood = orangeneighborhood
        self.__blueneighborhood = blueneighborhood


    #
    # Properties
    #

    @property
    def _id(self):
        return self.___id

    @_id.setter
    def _id(self, _id):
        self.___id = _id

    @property
    def redneighborhood(self):
        return self.__redneighborhood

    @redneighborhood.setter
    def redneighborhood(self, redneighborhood):
        self.__redneighborhood = redneighborhood

    @property
    def yellowneighborhood(self):
        return self.__yellowneighborhood

    @yellowneighborhood.setter
    def yellowneighborhood(self, yellowneighborhood):
        self.__yellowneighborhood = yellowneighborhood

    @property
    def purpleneigborhood(self):
        return self.__purpleneigborhood

    @purpleneigborhood.setter
    def purpleneigborhood(self, purpleneigborhood):
        self.__purpleneigborhood = purpleneigborhood

    @property
    def greenneighborhood(self):
        return self.__greenneighborhood

    @greenneighborhood.setter
    def greenneighborhood(self, greenneighborhood):
        self.__greenneighborhood = greenneighborhood

    @property
    def orangeneighborhood(self):
        return self.__orangeneighborhood

    @orangeneighborhood.setter
    def orangeneighborhood(self, orangeneighborhood):
        self.__orangeneighborhood = orangeneighborhood

    @property
    def blueneighborhood(self):
        return self.__blueneighborhood

    @blueneighborhood.setter
    def blueneighborhood(self, blueneighborhood):
        self.__blueneighborhood = blueneighborhood



    #
    # Methods
    #

    def __str__ (self):
        return ""



# Social_Bubbles.py -------------------------------------
from datetime import datetime, date

# 
# Python - Model Class - Quality_of_Life_Game.Social_Bubbles
# 
#
class Social_Bubbles(Object):

    #
    # Constructor
    #
    # Example: 
    # mySocial_Bubbles = Social_Bubbles( val1, val2,.. )
    #
    def __init__(self, _id = None, socioeconomicstatus = None, household = None, race = None, transportation = None, epidemiologicalfactors = None, healthfactors = None):
        self.___id = _id
        self.__socioeconomicstatus = socioeconomicstatus
        self.__household = household
        self.__race = race
        self.__transportation = transportation
        self.__epidemiologicalfactors = epidemiologicalfactors
        self.__healthfactors = healthfactors


    #
    # Properties
    #

    @property
    def _id(self):
        return self.___id

    @_id.setter
    def _id(self, _id):
        self.___id = _id

    @property
    def socioeconomicstatus(self):
        return self.__socioeconomicstatus

    @socioeconomicstatus.setter
    def socioeconomicstatus(self, socioeconomicstatus):
        self.__socioeconomicstatus = socioeconomicstatus

    @property
    def household(self):
        return self.__household

    @household.setter
    def household(self, household):
        self.__household = household

    @property
    def race(self):
        return self.__race

    @race.setter
    def race(self, race):
        self.__race = race

    @property
    def transportation(self):
        return self.__transportation

    @transportation.setter
    def transportation(self, transportation):
        self.__transportation = transportation

    @property
    def epidemiologicalfactors(self):
        return self.__epidemiologicalfactors

    @epidemiologicalfactors.setter
    def epidemiologicalfactors(self, epidemiologicalfactors):
        self.__epidemiologicalfactors = epidemiologicalfactors

    @property
    def healthfactors(self):
        return self.__healthfactors

    @healthfactors.setter
    def healthfactors(self, healthfactors):
        self.__healthfactors = healthfactors



    #
    # Methods
    #

    def __str__ (self):
        return ""



# Health_Factors.py -------------------------------------
from datetime import datetime, date

# 
# Python - Model Class - Quality_of_Life_Game.Health_Factors
# 
#
class Health_Factors(Object):

    #
    # Constructor
    #
    # Example: 
    # myHealth_Factors = Health_Factors( val1, val2,.. )
    #
    def __init__(self, _id = None, unexpectedpregnancy = None, measles = None, mumps = None, rubella = None, pneumonia = None, diabetes = None):
        self.___id = _id
        self.__unexpectedpregnancy = unexpectedpregnancy
        self.__measles = measles
        self.__mumps = mumps
        self.__rubella = rubella
        self.__pneumonia = pneumonia
        self.__diabetes = diabetes


    #
    # Properties
    #

    @property
    def _id(self):
        return self.___id

    @_id.setter
    def _id(self, _id):
        self.___id = _id

    @property
    def unexpectedpregnancy(self):
        return self.__unexpectedpregnancy

    @unexpectedpregnancy.setter
    def unexpectedpregnancy(self, unexpectedpregnancy):
        self.__unexpectedpregnancy = unexpectedpregnancy

    @property
    def measles(self):
        return self.__measles

    @measles.setter
    def measles(self, measles):
        self.__measles = measles

    @property
    def mumps(self):
        return self.__mumps

    @mumps.setter
    def mumps(self, mumps):
        self.__mumps = mumps

    @property
    def rubella(self):
        return self.__rubella

    @rubella.setter
    def rubella(self, rubella):
        self.__rubella = rubella

    @property
    def pneumonia(self):
        return self.__pneumonia

    @pneumonia.setter
    def pneumonia(self, pneumonia):
        self.__pneumonia = pneumonia

    @property
    def diabetes(self):
        return self.__diabetes

    @diabetes.setter
    def diabetes(self, diabetes):
        self.__diabetes = diabetes



    #
    # Methods
    #

    def __str__ (self):
        return ""



# Socioeconomic_Status.py -------------------------------------
from datetime import datetime, date

# 
# Python - Model Class - Quality_of_Life_Game.Socioeconomic_Status
# 2021-03-01 14:36:28
#
class Socioeconomic_Status(Object):

    #
    # Constructor
    #
    # Example: 
    # mySocioeconomic_Status = Socioeconomic_Status( val1, val2,.. )
    #
    def __init__(self, _id = None, bankruptcy = None, unemployment = None, homeinvasion = None, incomedrop = None, incomeloss = None, eviction = None):
        self.___id = _id
        self.__bankruptcy = bankruptcy
        self.__unemployment = unemployment
        self.__homeinvasion = homeinvasion
        self.__incomedrop = incomedrop
        self.__incomeloss = incomeloss
        self.__eviction = eviction


    #
    # Properties
    #

    @property
    def _id(self):
        return self.___id

    @_id.setter
    def _id(self, _id):
        self.___id = _id

    @property
    def bankruptcy(self):
        return self.__bankruptcy

    @bankruptcy.setter
    def bankruptcy(self, bankruptcy):
        self.__bankruptcy = bankruptcy

    @property
    def unemployment(self):
        return self.__unemployment

    @unemployment.setter
    def unemployment(self, unemployment):
        self.__unemployment = unemployment

    @property
    def homeinvasion(self):
        return self.__homeinvasion

    @homeinvasion.setter
    def homeinvasion(self, homeinvasion):
        self.__homeinvasion = homeinvasion

    @property
    def incomedrop(self):
        return self.__incomedrop

    @incomedrop.setter
    def incomedrop(self, incomedrop):
        self.__incomedrop = incomedrop

    @property
    def incomeloss(self):
        return self.__incomeloss

    @incomeloss.setter
    def incomeloss(self, incomeloss):
        self.__incomeloss = incomeloss

    @property
    def eviction(self):
        return self.__eviction

    @eviction.setter
    def eviction(self, eviction):
        self.__eviction = eviction



    #
    # Methods
    #

    def __str__ (self):
        return ""